proxy/http2/Http2Stream.h

```diff
@@ -124,6 +124,7 @@ class Http2Stream : public ProxyTransaction
   bool has_trailing_header() const;
   void set_request_headers(HTTPHdr &h2_headers);
   MIOBuffer *read_vio_writer() const;
+  bool update_read(int num_written);
```

Comment:
This name looks too generic. Please rename to `update_read_vio_ndone` or something else. Sorry for bothering you but we already have `update_read_request()`.